packages/sanity/src/core/hooks/useProjectSubscriptions.ts

```diff
@@ -0,0 +1,156 @@
+import {type SanityClient} from '@sanity/client'
+import {useMemo} from 'react'
+import {useObservable} from 'react-rx'
+import {type Observable, of} from 'rxjs'
+import {catchError, map, shareReplay, startWith} from 'rxjs/operators'
+
+import {useSource} from '../studio'
+import {DEFAULT_STUDIO_CLIENT_OPTIONS} from '../studioClient'
+import {useClient} from './useClient'
+
+type FeatureAttributes = Record<string, string | number | boolean | null>
+
+type Feature = {
```

Comment:
not critical, but we tend to prefer `interface` unless there's a specific reason to use `type`